test/index.js

```diff
@@ -23,6 +23,7 @@ var it = lab.it;
 
 
 var logPath = __dirname + '/test1.log';
+var flags = { flags: 'a' };
```

Comment:
No need to set this for every test. You'll need to find a way to run tests with different settings though.